protos/feast/serving/ServingService.proto

```diff
@@ -91,35 +91,58 @@ message GetOnlineFeaturesRequest {
     bool omit_entities_in_response = 3;
 
     message EntityRow {
-        // Request timestamp of this row. This value will be used, together with maxAge,
-        // to determine feature staleness.
+        // Request timestamp of this row. This value will be used, 
+        // together with maxAge, to determine feature staleness.
         google.protobuf.Timestamp entity_timestamp = 1;
 
         // Map containing mapping of entity name to entity value.
         map<string,feast.types.Value> fields = 2;
     }
 }
 
-message GetBatchFeaturesRequest {
-    // List of features that are being retrieved
-    repeated FeatureReference features = 3;
-
-    // Source of the entity dataset containing the timestamps and entity keys to retrieve
-    // features for.
-    DatasetSource dataset_source = 2;
-}
-
 message GetOnlineFeaturesResponse {
     // Feature values retrieved from feast.
     repeated FieldValues field_values = 1;
 
     message FieldValues {
         // Map of feature or entity name to feature/entity values.
         // Timestamps are not returned in this response.
-        map<string,feast.types.Value> fields = 1;
+        map<string, feast.types.Value> fields = 1;
+        // Map of feature or entity name to feature/entity statuses/metadata.
+        map<string, FieldStatus> statuses = 2;
+    }
+   
+    enum FieldStatus {
+        // Status is unset for this field.
+        INVALID = 0;
+
+        // Field value is present for this field and within maximum allowable range               
```

Comment:
This range is the age?

Reply:
Yes its max age. I have clarified in the documentation.